app/_src/networking/transparent-proxy/kubernetes.md

```diff
@@ -0,0 +1,265 @@
+{% enddanger %}
+
+#### Modifying control plane runtime configuration
+
+For instructions on modifying the control plane configuration, see the [Modifying the Configuration]({{ docs }}/documentation/configuration/#modifying-the-configuration) section in the [Control Plane Configuration]({{ docs }}/documentation/configuration/) documentation.
+
+#### Overriding list-based configuration settings
+
+For list-based settings like `runtime.kubernetes.injector.sidecarTraffic.excludeOutboundPorts`, only the value from the highest-precedence method will be applied; values from other methods will not be combined.
+
+For example, if `excludeOutboundPorts` is set in the control plane runtime configuration but an annotation like `traffic.kuma.io/exclude-outbound-ports` is used on a workload, the annotation will override the control plane setting, so only the ports in the annotation will be excluded.
+
+See the [**Order of Precedence**](#order-of-precedence) section for more details.
+
+<!-- vale Google.Headings = NO -->
+### Configuration in ConfigMap (experimental)
+<!-- vale Google.Headings = YES -->
+
+{% warning %}
+Because this feature impacts multiple underlying components, it is considered experimental. Use it with caution. {% if site.mesh_product_name == "Kuma" %} If you encounter any unexpected behavior or issues, please [**contact us**](/community) and [**submit an issue on GitHub**](https://github.com/kumahq/kuma/issues/new/choose). Your feedback is essential in helping us improve this feature. {% endif %}
+{% endwarning %}
+
+Until {{ Kuma }} 2.9, transparent proxy settings could only be modified through the [Control Plane Runtime Configuration](#control-plane-runtime-configuration) and [Annotations](#annotations), which had several limitations:
+
+- Not all settings were available through both methods; some could only be adjusted with annotations, while others were limited to the runtime configuration.
+- Control plane runtime settings were applied globally to all workloads with injected data planes, making it hard to customize settings for specific groups of workloads.
+- Annotation-based settings had to be added to each workload individually, making it difficult to manage the same configuration across multiple workloads.
+
+To address these issues, {{ Kuma }} now supports storing transparent proxy settings in dedicated ConfigMaps. These ConfigMaps can be applied at the namespace or individual workload level, offering greater flexibility and easier configuration.
+
+To enable this feature, set `{{ transparentProxy }}.configMap.enabled` during installation:
+
+{% cpinstall transparent-proxy-config-configmap-enabling %}
+{{ transparentProxy }}.configMap.enabled=true
+{% endcpinstall %}
+
+{% warning %}
+If you set `{{ transparentProxy }}.configMap.config` to an empty value, it will override `{{ transparentProxy }}.configMap.enabled` and disable the feature, even if `{{ transparentProxy }}.configMap.enabled` is set to `true`.
```

Comment:
this should just enable configMaps

Reply:
It was implemented like that, but with https://github.com/kumahq/kuma/pull/11988 it will just enable configMaps as you suggested

Reply:
wrapped this with `if_version` to not show it in 2.10.x+